### PR TITLE
Adding reproduction log, issue with Windows (32 bit), changing pom.xml and retrieval script

### PR DESCRIPTION
**Pull request by SyedHuq28**
Configuration:
Processor Speed (Max): 1800 MHz
Memory (RAM): 16 GB
Processor: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (GenuineIntel)
Operating System: Microsoft Windows 10 Pro (64-bit)
Number of Cores: 4
Number of Logical Processors: 8

Initially, I tried to solve the problems associated with the repo Anserini using PowerShell and converting the scripts to PowerShell format for Windows. However, I encountered a few issues while running on Windows. For instance, installing Maven and trec_eval on Windows resulted in errors. Consequently, I resorted to virtualization on my computer using WSL (Ubuntu 22.04.4). One issue I encountered while using Ubuntu was that when I extracted any compressed files from the Ubuntu terminal, later on, when I attempted to read or write to that file, it denied permission. This occurred because my Ubuntu terminal account had associated passwords and any extracted files required permission to be accessed. Therefore, I had to manually extract compressed files to avoid read/write permission issues rather than using a script for extraction.

Besides, my Windows was running in a 32-bit configuration, whereas the structure of trec_eval was designed for 64-bit systems. Despite this hurdle and other dependency installation and usage problems, I solved them using WSL (Ubuntu 22.04.4).

Specifications of WSL:
Ubuntu 22.04.4
Linux 5.15.146.1
Architecture: microsoft-standard-WSL2 x86_64
OpenJDK 11.0.22

Another major issue that I encountered and resolved was in the retrieval script mentioned in experiments-msmarco-passage.md and pom.xml file. I had to modify some lines to make everything run perfectly.

In summary, I initially attempted tasks on Windows primarily using PowerShell terminal in VS Code. After encountering some errors and making some progress, I completed the remaining tasks using WSL (Ubuntu). With the aforementioned changes and tweaks, I was able to achieve the expected results and reproduce all of the results from the given assignment.

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ target/appassembler/bin/SearchCollection \
   -topicReader TsvInt \
   -output runs/run.msmarco-passage.dev.small.tsv -format msmarco \
   -parallelism 4 \
-  -bm25 -bm25.k1 0.82 -bm25.b 0.68 -hits 1000
+  -bm25 "-bm25.k1" 0.82 "-bm25.b" 0.68 -hits 1000
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:123]:
```suggestion
  -bm25 -bm25.k1 0.82 -bm25.b 0.68 -hits 1000
```

File: pom.xml [hunk 2]
```diff
@@ -284,32 +284,29 @@
...
-          <execution>
-            <id>make-assembly</id> <!-- this is used for inheritance merges -->
-            <phase>package</phase> <!-- bind to the packaging phase -->
-            <goals>
-              <goal>single</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
+       <plugin>
```
Comment (lintool) [inline pom.xml:287]: please revert changes.

**Review by lintool — CHANGES_REQUESTED** [review]
When I go to your URL: https://github.com/castorini/anserini/commit/fb8a813ac56fea21659d7465aeadc590d619ce5e

I get the error: This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Please fix. Should refer to a commit point on master.

**Comment by lintool** [comment]
https://github.com/castorini/anserini/discussions/2428

**Comment by SyedHuq28** [comment]
Apologies for pushing the changes directly to the repo and also for sending commit that belongs to a fork outside the repository, I am again opening a fresh pull request with the suggested changes
